wbstack/src/Settings/LocalSettings.php

```diff
@@ -451,11 +463,13 @@ function onPersonalUrlsConfirmAccount( array &$personal_urls, Title $title, Skin
 $wgReCaptchaSecretKey = getenv('MW_RECAPTCHA_SECRETKEY');
 
 # Mailgun
-wfLoadExtension( 'Mailgun' );
-$wgMailgunAPIKey = getenv('MW_MAILGUN_API_KEY');
-$wgMailgunDomain = getenv('MW_MAILGUN_DOMAIN');
-// Example Endpoint "https://api.mailgun.net"
-$wgMailgunEndpoint = getenv('MW_MAILGUN_ENDPOINT');
+if (getenv('MW_MAILGUN_ENABLED') === 'yes') {
```

Comment:
By default right now this will mean that the extension is no loaded at image building time, which will result in the localization cache for this extension not being built into the image.
This likely needs to be set by default to yes in the image until such a time that wbstack.com doesnt need it